workbench/tests/renderer/app.test.js

```diff
@@ -110,6 +110,9 @@ describe('Various ways to open and close InVEST models', () => {
     const setupTab = await findByText('Setup');
     expect(setupTab.classList.contains('active')).toBeTruthy();
     expect(getSpec).toHaveBeenCalledTimes(1);
+    const navTab = await findByRole('tab', { name: MOCK_MODEL_TITLE });
+    userEvent.hover(navTab);
+    getByTestId('tab-tooltip');
```

Comment:
Does the `Tooltip` element have role `tooltip`? https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tooltip_role

Basically I'm wondering if we can avoid using a test ID to find the element, as that would make the test a bit more resilient during refactors.

Along those lines...Tooltips often show up with a delay. I don't think you have one set here, but changing from `getBy` to ` await findBy` would allow us to adjust the delay at any point in the future and not break the test.

Reply:
Good idea, the `tooltip` role seems to work. I did try to avoid using a test ID, but I couldn't find it documented anywhere what the role of the overlay is, and inspecting it in the browser didn't work.